frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -957,9 +970,55 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// Submit a signed solution during the emergency phase.
+		/// It will go through the `feasibility_check` right away.
+		///
+		/// The dispatch origin for this call must be __signed__.
+		///
+		/// The deposit that is reserved might be rewarded or slashed based on
```

Comment:
these comments are wrapped at 80? please make sure you are wrapping your lines at 100, no more or no less.

Reply:
https://marketplace.visualstudio.com/items?itemName=stkb.rewrap